great_tables/_utils_render_latex.py

```diff
@@ -295,49 +295,38 @@ def create_columns_component_l(data: GTData) -> str:
         #     ids=True,
         #     omit_columns_row=True,
         # )
-
-        for i in range(len(spanners)):
-            spanners_row = spanners[i]
-
-            for k, v in spanners_row.items():
-                if v is None:
-                    spanners_row[k] = ""
```

Comment:
Just noting this mutation that was fixed to not mutate (hopefully isn't leaned on elsewhere in code)

Reply:
Could you expand on this? I am not sure I understand what you mean